lib/request-handler.js

```diff
@@ -186,14 +186,11 @@ module.exports = function processRequest (options, request, response) {
 
         resultStream.once('error', handleError);
 
-        if (Array.isArray(parsedTemplate)) {
-            parsedTemplate.forEach((item) => {
-                resultStream.write(item);
-            });
-            resultStream.end();
-        } else {
-            resultStream.end(parsedTemplate);
-        }
+        parsedTemplate.forEach((item) => {
```

Comment:
@vigneshshanmugam Would it make sense to add a test or a task to add test for this module?

Reply:
Its already there in tests/parse-template.. since we never reached this part, codecov was decreased. I found this only through codecov report.